src/LineUI/LineSet.tsx

```diff
@@ -229,13 +229,15 @@ const LineSet : React.FC<ILineSetProps> = ({ getCTM, boundaries, unit, size, lin
       styling={styling}
       lineMoveCallback={lineDragUpdate}
       lineMoveStartCallback={lineDragStart}
+      bothDirectionMark={!!lineSetData.showSmallDirectionMark}
```

Comment:
It seems !! is wrong and also implementation of getting value of bothDirectionMark is wrong.
 
I think when we have **showDirectionMark** and **showSmallDirectionMark** both true then this **bothDirectionMark** should be true otherwise it should be false.

Reply:
Done

Reply:
Its not done. Check comments again and fix it.